SWLOR.Game.Server/Feature/RecipeDefinition/FabricationRecipeDefinition/FurnitureRecipes.cs

```diff
@@ -297,1943 +297,2434 @@ private void Tier1()
+                .Resref("structure_0154")
+                .Level(30)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 3)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("elec_good", 6)
+                .Component("ref_plagionite", 3);
+
+            // Pot, Plant, Aloa
+            _builder.Create(RecipeType.PotPlantAloa, SkillType.Fabrication)
+                .Category(RecipeCategoryType.MiscellaneousFurniture)
+                .Resref("structure_0187")
+                .Level(30)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 3)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("ancient_wood", 6)
+                .Component("lth_good", 3);
+
+            // Bar White
+            _builder.Create(RecipeType.BarWhite, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Surfaces)
+                .Resref("structure_0229")
+                .Level(25)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 3)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("ref_plagionite", 5)
+                .Component("fiberp_good", 3);
+
+            // Bathtub
+            _builder.Create(RecipeType.Bathtub, SkillType.Fabrication)
+                .Category(RecipeCategoryType.MiscellaneousFurniture)
+                .Resref("structure_0232")
+                .Level(20)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 3)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("fiberpl_good", 4)
+                .Component("ref_plagionite", 2);
+
+            // Store Counter (Stained)
+            _builder.Create(RecipeType.StoreCounterStained, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Surfaces)
+                .Resref("structure_0236")
+                .Level(23)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 3)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("ref_plagionite", 4)
+                .Component("fiberpl_good", 2);
+
+            // Coffee Maker
+            _builder.Create(RecipeType.CoffeeMaker, SkillType.Fabrication)
+                .Category(RecipeCategoryType.Electronics)
+                .Resref("structure_0227")
+                .Level(21)
+                .Quantity(1)
+                .RequirementPerk(PerkType.FurnitureBlueprints, 2)
+                .EnhancementSlots(RecipeEnhancementType.Structure, 2)
+                .Component("fiberpl_flawed", 2)
+                .Component("elec_flawed", 3);
+
+            // Television, Old Model
+            _builder.Create(RecipeType.CoffeeMaker, SkillType.Fabrication)
```

Comment:
This is the wrong enumeration for the Television, Old Model. Please correct it with the right enum.